examples/linear/README.md

```diff
@@ -0,0 +1,5 @@
+# Linear Application Example
+
+This example assumes you already have EveryAuth configured in your development environment. In case you don’t, follow the [configuration steps](https://github.com/fusebit/everyauth-express#getting-started).
+
+More details coming after blog post.
```

Comment:
Let's go ahead and flush those out here, at least to the level that the sfdc and others are.